vsc_pdf_template_transformer/lib/models/tpl_annotation_circle.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:pdf/widgets.dart';
+import 'package:vsc_pdf_template_transformer/utils/evaluator.dart';
+import '../utils/annotation_builder.dart' as ab;
+
+part 'tpl_annotation_circle.g.dart';
+
+@JsonSerializable(
+  checked: true,
+  disallowUnrecognizedKeys: false,
+  explicitToJson: true,
+)
+class TplAnnotationCircle implements ab.AnnotationBuilder {
```

Comment:
I specifically didn't include the Annotation classes in the milestones, or lists of tasks on Trello. Why did you build all of these out? I understand that TextSpan _can_ take an AnnotationBuilder, but it's not required and we should have left it null. If you have questions about whether you should spend time on something in the future, please ask.